src/components/expandableSection/index.tsx

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import React, {useState} from 'react';
+import {LayoutAnimation} from 'react-native';
+import View from '../view';
+//@ts-ignore
+import ListItem from '../listItem';
+import Button from '../button';
+
+
+export type ExpandableSectionProps = {
+  /**
+   * expandableSection header element
+   */
+  sectionHeader?: JSX.Element;
+  /**
+   * expandableSection expandable children
+   */
+  children?: React.ReactNode;
+  /**
+   * expandableSection icon color
+   */
+  iconColor?: string;
+};
+
+const chevronDown = require('../../assets/icons/chevronDown.png');
+const chevronUp = require('../../assets/icons/chevronUp.png');
+
+function ExpandableSection(props: ExpandableSectionProps) {
+  const [expanded, setExpanded] = useState(false);
+  const {iconColor, sectionHeader, children} = props;
+
+  const onExpand = () => {
+    LayoutAnimation.configureNext(LayoutAnimation.Presets.spring);
```

Comment:
I recommend toning down a little the spring animation (: 
The UX who'll review it will probably comment about it.. 

If you want a more specific specification on the animation talk to Or.
Maybe she can give some reference for how it should look.